docs/FAQ.md

```diff
@@ -76,6 +76,23 @@ We think of Zulip Terminal as a keyboard-centric client. Consequently, while fun
 
 If any of the hotkeys don't work for you, feel free to open an issue or discuss it on [#zulip-terminal](https://chat.zulip.org/#narrow/stream/206-zulip-terminal).
 
+## Are there any themes available, other than the default one?
```

Comment:
This is great :+1: I cherry-picked this commit and merged it :tada: I moved the section further up in the file as it's a higher priority than some others, and also made some other minor textual adjustments.